vsphere/alerts.libsonnet

```diff
@@ -0,0 +1,92 @@
+{
+  new(this): {
+
+    groups: [
+      {
+        name: this.config.uid,
+        rules: [
+          {
+            alert: 'VSphereHostInfoCpuUtilization',
```

Comment:
Capitalization for the alert names doesn't match the README

Reply:
Yea for some reason the linter asks for camel case, but it doesnt accept camel case, it wants the v to be capitalized. 🤦 . So i just ended up capitalizing it all. I've updated the readme